dts/bindings/ethernet/ethernet.yaml

```diff
@@ -21,3 +22,8 @@ properties:
       It is driver specific how the OUI octets are handled.
 
       If set we ignore any setting of the local-mac-address property.
+
+  phy-handle:
```

Comment:
Non blocking. I still thinking that this is a good opportunity to rename the file `ethernet.yaml` to `ethernet-controller.yaml` as a preparation to property rename. Otherwise project will be always in debt and structural changes in future could be more difficult since code base is growing.

Reply:
Hi @nandojve I’m curious, is there any umbrella issue where property rename to follow Linux bindings is being discussed?

Reply:
Hi @manuargue , not that I have knowledge. The policy in general for networking is being close to Linux. Zephyr has their own autonomy but in this case being equal in the filename and property makes sense. This changed was motivated by #60816.

Reply:
Cool with that, added a patch to rename the file.